backend/src/services/preferences/wishlist.service.ts

```diff
@@ -0,0 +1,43 @@
+import { prisma } from '@data/prisma-client';
+
+import type { WishlistResponseDto } from '@autoline/shared';
+import type { WishlistInput } from '@common/types/types';
+
+const setWishlist = async (
+  input: WishlistInput,
+): Promise<WishlistResponseDto> => {
+  const { user_id, model_id, complectation_id } = input;
```

Comment:
We **must** use only **`camelCase`** names for the variables in the JS code. Please, don't use `snake_case` when it's possible